ios-trivia-game/API/FirebaseClient.swift

```diff
@@ -57,6 +57,16 @@ class FirebaseClient {
         }
     }
     
+    // updates current player count of game
+    func updatePlayerCount(roomId: String, change: Int) {
+        let gamePath = "\(Constants.GAME_ROOM_TABLE_NAME)/\(roomId)/current_num_players"
+        self.ref.child(gamePath).runTransactionBlock { (data) -> FIRTransactionResult in
```

Comment:
In that case, might be better to just use the transaction block then. Since multiple people might've joined after the request is submitted.